src/data/hardware.ts

```diff
@@ -0,0 +1,19 @@
+export const BOARD_NAMES: Record<string, string> = {
```

Comment:
We should sync this with https://github.com/home-assistant/analytics.home-assistant.io/blob/dev/site/src/analytics-os-boards.ts#L6-L24

Reply:
```suggestion
// Keep in sync with https://github.com/home-assistant/analytics.home-assistant.io/blob/dev/site/src/analytics-os-boards.ts#L6-L24
export const BOARD_NAMES: Record<string, string> = {
```